executor/show.go

```diff
@@ -1485,6 +1493,17 @@ func (e *ShowExec) fetchShowCreatePlacementPolicy() error {
 	return nil
 }
 
+// fetchShowCreateResourceGroup composes show create resource group result.
+func (e *ShowExec) fetchShowCreateResourceGroup() error {
+	group, found := e.is.ResourceGroupByName(e.ResourceGroupName)
+	if !found {
+		return infoschema.ErrResourceGroupNotExists.GenWithStackByArgs(e.ResourceGroupName.O)
+	}
+	showCreate := ConstructResultOfShowCreateResourceGroup(group)
```

Comment:
The function is not used by other places, so there is no need to expose it, `constructResultOfShowCreateResourceGroup` would be better.